health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -161,89 +161,84 @@ const FormulaConfiguration = ({ onSelect, category, customProps, formulas: initi
-                    t={t}
-                    isMandatory={true}
-                    option={inputOptions}
-                    select={(value) => {
-                      handleFormulaChange(formula.output, "input", value, category);
-                    }}
-                    selected={() => ({ code: formula.input })}
-                    optionKey="code"
-                    showToolTip={true}
-                    style={{ width: "20rem" }}
-                  />
+              <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)